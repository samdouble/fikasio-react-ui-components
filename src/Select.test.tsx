import React from 'react';
import { render } from '@testing-library/react';
import { Select } from '.';

describe('Select', () => {
  it('Renders correctly', () => {
    const { baseElement } = render(
      <Select
        options={[
          { label: 'Option 1', value: '1' },
          { label: 'Option 2', value: '2' },
        ]}
      />
    );
    expect(baseElement).toMatchSnapshot();
  });
});
