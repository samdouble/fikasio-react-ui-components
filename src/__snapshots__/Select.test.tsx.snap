// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Select Renders correctly 1`] = `
<body>
  <div>
    <span
      class="fikasio-select fikasio-theme-dark"
    >
      <div
        class="MuiFormControl-root css-hs8t5g-MuiFormControl-root"
      >
        <div
          class="MuiInputBase-root MuiInputBase-colorPrimary MuiInputBase-formControl MuiInputBase-sizeSmall css-1swlhp1-MuiInputBase-root"
        >
          <div
            aria-controls=":r0:"
            aria-expanded="false"
            aria-haspopup="listbox"
            class="MuiSelect-select MuiSelect-outlined MuiInputBase-input MuiInputBase-inputSizeSmall css-id4s6h-MuiSelect-select-MuiInputBase-input"
            role="combobox"
            tabindex="0"
          >
            Option 1
          </div>
          <input
            aria-hidden="true"
            aria-invalid="false"
            class="MuiSelect-nativeInput css-j0riat-MuiSelect-nativeInput"
            tabindex="-1"
            value="1"
          />
          <svg
            aria-hidden="true"
            class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium MuiSelect-icon MuiSelect-iconOutlined css-lohd6h-MuiSvgIcon-root-MuiSelect-icon"
            data-testid="ArrowDropDownIcon"
            focusable="false"
            viewBox="0 0 24 24"
          >
            <path
              d="M7 10l5 5 5-5z"
            />
          </svg>
        </div>
      </div>
    </span>
  </div>
</body>
`;
